### PR TITLE
Fixed api blueprint example

### PR DESCRIPTION
**Pull request by sr-leann**
* **Please check if the PR fulfills these requirements**
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Is there any issue related to this PR in other repository?** (such as dojot/dojot)
* dojot/dojot#1766

* **Other information**:

### CODE REVIEW

File: docs/api.apib [hunk 1]
```diff
@@ -21,7 +21,7 @@ send events to the data broker or execute a given javascript function.
     + Body
 
             {
-                "time": "0/5 * * * *",
+                "time": "*/5 * * * *",
```
Comment (rascaraficci) [inline docs/api.apib]: If I'm not wrong, the cron's expressions accepted by the library we use starts on second not minute. So, the right expression should be `"0 */5 0 0 0 0 "`

File: docs/api.apib [hunk 2]
```diff
@@ -185,7 +185,7 @@ send events to the data broker or execute a given javascript function.
...
```
Comment (rascaraficci) [inline docs/api.apib]: The same comment above.

**Review by rascaraficci — CHANGES_REQUESTED** [review]
I left a comment!

File: docs/api.apib (hunk 1 above)
Comment (sr-leann) [reply docs/api.apib]: Leaving it off, the default will be 0, but for the sake of understanding the possibility of using seconds, I will add the seconds field.

**Review by rascaraficci — APPROVED** [review]
It looks OK!

**Review by mprevide — APPROVED** [review]

**Comment by mprevide** [comment]
Add this changes at https://github.com/dojot/cron/pull/43

**Pull request merged**